concent_api/conductor/models.py

```diff
@@ -90,6 +107,9 @@ class UploadReport(Model):
     # Foreign key to VerificationRequest. Can be NULL if there's no corresponding request.
     verification_request = ForeignKey(VerificationRequest, related_name='upload_reports', blank=True, null=True)
 
+    # Foreign key to z. Can be NULL if there's no corresponding request.
+    result_transfer_request = ForeignKey(ResultTransferRequest, related_name='upload_reports', blank=True, null=True)
+
     # Indicates when conductor has been notified about the upload.
     created_at      = DateTimeField(default=timezone.now)
```

Comment:
You can fix style in here I think. It's the last place in this file with our old one.

Reply:
:+1: done